Alex_RTOS_Sailboat/RTOS_framework_v1/src/sail_ctrl.c

```diff
@@ -314,7 +313,7 @@ void process_heading_readings(void)
 
 
 
-void assign_weatherstation_readings(void) {
+/*void assign_weatherstation_readings(void) {
```

Comment:
Remove comment

Reply:
Comment removed. 
P.S. - This method will need to be removed after getting rid of the weather sensor source files